lib/server.js

```diff
@@ -192,6 +192,14 @@ function handleRequest (opts, req, resp) {
     };
     reqOpts.log = reqOpts.logger.log.bind(reqOpts.logger);
 
+    // Simplistic count of requests from public & private networks
+    var remoteAddr = req.socket.remoteAddress;
```

Comment:
`remoteAddress` will report the connecting *client* which will be the LVS (or Varnish). In that respect, you are right to count `127.x.x.x`, but `10.x.x.x` will cover all of the remaining requests, which means we'll have 100% of private requests.

Varnish sets the `X-Forwarded-For` header, so we should use that one, and, if it's missing, use the socket remote address.